README.rst

```diff
@@ -80,13 +84,15 @@ You can also get the current ``FiscalQuarter`` with:
 
 .. code-block:: python
 
+   >>> from fiscalyear import *
```

Comment:
I think you can remove this line, the README already assumes that was imported at the top.

Reply:
It came through when I copied and pasted the content from basic_usage.rst. Figured I'd leave it in and see what you felt. Happy to remove it though.

Reply:
Can you make `basic_usage.rst` the same? We should only import things once.